pkgs/misc/kitty-themes/default.nix

```
@@ -0,0 +1,28 @@
+{
+  fetchFromGitHub, lib, stdenv
+}:
+
+stdenv.mkDerivation rec {
+  pname = "kitty-themes";
+  version = "2022-01-01";
+
+  src = fetchFromGitHub {
+    owner = "kovidgoyal";
+    repo = pname;
+    rev = "76e0c1151ea2d48a6c9c3842095c9781051e5b88";
+    sha256 = "8RIZ9ytjfWVAWvUQPORL1EDS0ncFSzq7e5kxxPdet3g=";
+  };
+
+  installPhase = ''
+    mkdir -p $out
+    mv themes/*.conf $out
+  '';
+
+  meta = with lib; {
+    homepage = "https://github.com/kovidgoyal/kitty-themes";
+    description = "Themes for the kitty terminal emulator";
+    license = licenses.gpl3Only;
+    platforms = platforms.darwin ++ platforms.linux;
```

One last suggestion from me:
```suggestion
    platforms = platforms.all;
```
Since this package contains no executable code, it should build on every architecture.